frontend/src/views/OrderView.vue

```diff
@@ -1,74 +1,183 @@
-<script setup>
+<script setup lang="ts">
+import AutoComplete from "primevue/autocomplete";
 import Button from "primevue/button";
+import Card from "primevue/card";
 import Dropdown from "primevue/dropdown";
 import FloatLabel from "primevue/floatlabel";
+import InputNumber from "primevue/inputnumber";
 import InputText from "primevue/inputtext";
 import Textarea from "primevue/textarea";
-import { ref } from "vue";
+import { computed, onMounted, ref } from "vue";
+import { useRoute, useRouter } from "vue-router";
 
+import { createOrder, getClients } from "@/backendClient";
 import { OrderStatus } from "@/global/types/appTypes";
+import type { OrderCreate, OrderUpdate } from "@/global/types/dataEditTypes";
+import type { Client } from "@/global/types/entities";
+import { ROUTES } from "@/router";
+
+interface ExtendedClient extends Client {
+  full_name?: string;
+}
```

Comment:
Does it need a new interface for concatenating the name?

Wouldn't a computed property be enough?